components/brave_welcome_ui/theme/index.ts

```diff
@@ -0,0 +1,8 @@
+import styled, * as Styled from 'styled-components'
+import IBraveTheme from 'brave-ui/theme/theme-interface'
+import IWelcomeTheme from './welcome-theme'
+
+type Theme = IBraveTheme & IWelcomeTheme
+
+export default styled as unknown as Styled.ThemedStyledInterface<Theme>
```

Comment:
yes, it allows the type to be cast, otherwise you cannot directly cast `styled` to include the theme. But we know that it does include the specific Welcome theme, so it's ok.

This is the suggested way to have multiple themes in a project. Otherwise it will just assume our re-defined DefaultTheme.